src/plugins/saved_objects_management/public/management_section/objects_table/components/header.tsx

```diff
@@ -141,9 +141,12 @@ export const Header = ({
           run: onExportAll,
           controlType: 'button',
           iconType: 'exportAction',
-          label: i18n.translate('savedObjectsManagement.objectsTable.header.exportButtonLabel', {
-            defaultMessage: 'Export all objects',
-          }),
+          label: i18n.translate(
+            'savedObjectsManagement.objectsTable.header.exportAssetsButtonLabel',
+            {
+              defaultMessage: 'Export all assets',
+            }
+          ),
```

Comment:
do we need to check new home page flag for all those wording change?

Reply:
This new home page flag check logic is already in line 121. I think we don't need to do it again here. This "Export " button only displayed when new home page flag enabled.